src/createServer.js

```diff
@@ -1,3 +1,59 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const allCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+    const normalizedUrl = new URL(req.url, `http//${req.headers.host}`);
+    const text = normalizedUrl.pathname.slice(1);
+    const entries = Object.fromEntries(normalizedUrl.searchParams.entries());
+    const toCase = entries.toCase;
+    const errors = [];
+
+    if (!text) {
+      errors.push({
+        message: 'Text to convert is required. '
+          + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".'
```

Comment:
Do not rely on the unknown string, make constants for this or create a special object with all messages. It will be good for future reusability.